src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -135,7 +135,7 @@
             { 'openSUSEs OBS Portal' => 'http://en.opensuse.org/Portal:Build_service' },
             { 'Reporting a Bug'   => 'http://openbuildservice.org/support/' } ]
 
-          @helplinks << { 'Terms of Service' => announcement_path(Announcement.last) } if Announcement.last.present?
+          @helplinks << { 'Announcements' => announcement_path(Announcement.last) } if Announcement.last.present?
```

Comment:
:thought_balloon:  I would use it in singular: "Announcement". As it is now, we always show only one at a time.